scanner/generate_byte_code.py

```diff
@@ -0,0 +1,44 @@
+import click
+
+from solc import compile_source
+
+
+def convert_to_byte_code(file, contract_name):
+    report = {}
+    compiled_sol = compile_source(open(file, mode='r').read())
+    if type(contract_name) is list:
+        for contract in contract_name:
+            contract_interface = compiled_sol['<stdin>:{}'.format(str(contract))]
+            report[str(contract)] = {'bytecode': contract_interface['bin'], 'abi': compiled_sol['<stdin>:' +
+                                                                                                str(contract)]['abi']}
+        print(report)
+    else:
+        contract_interface = compiled_sol['<stdin>:{}'.format(str(contract_name))]
+        report[str(contract_name)] = {'bytecode': contract_interface['bin'], 'abi': compiled_sol['<stdin>:' +
+                                                                                                 str(contract_name)]
+                                                                                                ['abi']}
+        print(report)
+
+
+def get_contract_names(file):
+    path_to_file = open(file, mode='r')
+    compiled_sol = compile_source(path_to_file.read())
+
+    name_of_contract = compiled_sol['<stdin>:SafeMath']['ast']['attributes']['exportedSymbols']
```

Comment:
Looks strange that we hardcode **SafeMath** here: `compiled_sol['<stdin>:SafeMath']`